docs/azure/ai/get-started-app-chat-template.md

```diff
@@ -0,0 +1,300 @@
+
+    :::image type="content" source="../media/get-started-app-chat-template/browser-chat-initial-answer.png" alt-text="Screenshot of chat app's first answer.":::
+
+1. From the answer, select a citation. A pop-up window will open displaying the source of the information.
+
+    :::image type="content" source="../media/get-started-app-chat-template/browser-chat-initial-answer-citation-highlighted.png" alt-text="Screenshot of chat app's first answer with its citation highlighted in a red box.":::
+
+1. Navigate between the tabs at the top of the answer box to understand how the answer was generated.
+
+    |Tab|Description|
+    |---|---|
+    |**Thought process**|This is a script of the interactions in chat. You can view the system prompt (`content`) and your user question (`content`).|
+    |**Supporting content**|This includes the information to answer your question and the source material. The number of source material citations is noted in the **Developer settings**. The default value is **3**.|
+    |**Citation**|This displays the original page that contains the citation.|
+
+1. When you are done, navigate back to the answer tab.
+
+### Use chat app settings to change behavior of responses
+
+The intelligence of the chat is determined by the OpenAI model and the settings that are used to interact with the model.
+
+:::image type="content" source="../media/get-started-app-chat-template/browser-chat-developer-settings-chat-pane.png" alt-text="Screenshot of chat developer settings.":::
+
+|Setting|Description|
+|---|---|
+|Override prompt template|This is the prompt that is used to generate the answer.|
+|Retrieve this many search results|This is the number of search results that are used to generate the answer. You can see these sources returned in the _Thought process_ and _Supporting content_ tabs of the citation. |
+|Exclude category|This is the category of documents that are excluded from the search results.|
+|Use semantic ranker for retrieval|This is a feature of [Azure Cognitive Search](/azure/search/semantic-search-overview#what-is-semantic-search) that uses machine learning to improve the relevance of search results.|
+|Retrieval mode|**Vectors + Text** means that the search results are based on the text of the documents and the embeddings of the documents. **Vectors** means that the search results are based on the embeddings of the documents. **Text** means that the search results are based on the text of the documents.|
+|Use query-contextual summaries instead of whole documents| Limits the query to only search summarized data instead of a more thorough search of the full document. |
+|Suggest follow-up questions|Have the chat app suggest follow-up questions based on the answer.|
+
+The following steps walk you through the process of changing the settings.
+
+1. In the browser, select the gear icon in the upper right of the page.
+1. Check the **Suggest follow-up questions** checkbox and ask the same question again.
+
+    ```
```

Comment:
💡 ProTip! Even though this isn't a valid language slug, we know that it will render as a pass thru - letting us put anything in there we want. Since we have control of it, we could do either "Text" or "Input".

```suggestion
    ```Text
```